packages/clerk-js/src/core/clerk.ts

```diff
@@ -710,8 +710,10 @@ export default class Clerk implements ClerkInterface {
   };
 
   #resetComponentsState = () => {
```

Comment:
FYI: This is called by setSession and should work both for full and headless variant.

Reply:
Great catch

Reply:
Moving forward we should probably discuss how we can abstract all the small differences between web and native platforms await from the `Clerk` singleton